Index: lesson3/task1.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Пользователь вводит предложение, заменить все пробелы на \"-\" 2-мя способами\r\ntext = input(\"введите любое предложение \")\r\n\r\n# первый способ\r\n\r\nprint(text.replace(' ', '-'))\r\n\r\n\r\n# второй способ\r\n\r\n# text = text.split(' ')\r\n# text = '-'.join(text)\r\n# print(text)\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/lesson3/task1.py b/lesson3/task1.py
--- a/lesson3/task1.py	(revision eb251bf883453914259074eb0a844ed8548b4aa8)
+++ b/lesson3/task1.py	(date 1706952082610)
@@ -3,11 +3,11 @@
 
 # первый способ
 
-print(text.replace(' ', '-'))
+# print(text.replace(' ', '-'))
 
 
 # второй способ
 
-# text = text.split(' ')
-# text = '-'.join(text)
-# print(text)
+text = text.split(' ')
+text = '-'.join(text)
+print(text)
Index: lesson3/task3.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Пользователь вводит Имя, Возраст и Город, сформировать приветственное сообщение путем форматирования 3-мя способами\r\nname = input(\"введите имя \")\r\nage = input(\"введите возраст \")\r\ncity = input(\"введите ваш город \")\r\nage = int(age)\r\n# первый способ\r\nprint(f\"здравствуйте,{name}, вам {age} лет и вы проживате в городе {city}\")\r\n\r\n# второй способ\r\n# print('здравствуйте, {}, вам {} лет и вы проживате в городе {}'.format(name, age, city))\r\n\r\n# третий способ\r\n# print('здравствуйте, %s, вам %d лет и вы проживате в городе %s'%(name, age, city))\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/lesson3/task3.py b/lesson3/task3.py
--- a/lesson3/task3.py	(revision eb251bf883453914259074eb0a844ed8548b4aa8)
+++ b/lesson3/task3.py	(date 1706952016929)
@@ -5,7 +5,7 @@
 age = int(age)
 # первый способ
 print(f"здравствуйте,{name}, вам {age} лет и вы проживате в городе {city}")
-
+`
 # второй способ
 # print('здравствуйте, {}, вам {} лет и вы проживате в городе {}'.format(name, age, city))
 
